Jenkinsfile

```diff
@@ -167,6 +167,7 @@ stage('Build') {
           sh "${docker_run} ${ci_cpu} ./tests/scripts/task_python_vta_tsim.sh"
           sh "${docker_run} ${ci_cpu} ./tests/scripts/task_golang.sh"
           sh "${docker_run} ${ci_cpu} ./tests/scripts/task_rust.sh"
+          sh "${docker_run} ${ci_cpu} ./tests/scripts/task_python_tvmc.sh"
```

Comment:
NOTE: changes to the Jenkinsfile won't be immediately reflected, for now, set the sh file to a blank file, and then we can turn in on as a followup PR

Reply:
with the new approach, `tvmc` tests will be embedded on `tvm` python  tests.